cmd/watch.go

```diff
@@ -97,7 +97,9 @@ func delveInitFile(exec *testjson.Execution) (string, func(), error) {
 	for _, tc := range exec.Failed() {
 		fmt.Fprintf(buf, "break %s\n", tc.Test.Name())
 	}
-	buf.WriteString("continue\n")
+	if _, err := buf.WriteString("continue\n"); err != nil {
+		panic(err)
+	}
```

Comment:
This `buf` here is a `bufio.Writer` which has a really cool feature: https://golang.org/pkg/bufio/#Writer

>  If an error occurs writing to a Writer, no more data will be accepted and all subsequent writes, and Flush, will return the error

What this allows us to do is perform multiple writes (or in this case `WriteString`) and ignore the error. After this line we `Flush()` the buffer, and handle the error.  So even though we aren't handling the error on this line, we do correctly handle any error this write may have performed in the call to Flush below.

The linter doesn't complain, but our `bufio.Writer` is actually capture errors for all the `fmt.Fprintf` calls in the loop above as well. The linter defaults to ignoring errors from `Fprintf` (for some reason, I'm not exactly sure why).

I think this is a really nice pattern for error handling. I've seen it mentioned in an article by Rob Pike (although sadly I can't find that link right now), [this article from Dave Cheney](https://dave.cheney.net/2019/01/27/eliminate-error-handling-by-eliminating-errors) goes into a bit as well.